evalml/tests/data_checks_tests/test_datetime_format_data_check.py

```diff
@@ -12,11 +13,18 @@
 
 @pytest.mark.parametrize("input_type", ["pd", "ww"])
 @pytest.mark.parametrize(
-    "uneven,type_errors", [(True, False), (False, True), (False, False)]
+    "redundant,missing,uneven,type_errors",
```

Comment:
Any reason to not just parametrize over each individually?  I think it makes the failures look a little more obvious.